Fix failing sonar check on PRs

Right now sonar check is failing which is creating lots of confusion.

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Fabric8 Maven :: Parent ............................ SUCCESS [  0.684 s]
[INFO] Fabric8 Maven :: Core .............................. SUCCESS [ 32.793 s]
[INFO] Fabric8 Maven :: Enricher :: API ................... SUCCESS [  2.138 s]
[INFO] Fabric8 Maven :: Enricher :: Standard .............. SUCCESS [  9.977 s]
[INFO] Fabric8 Maven :: Enricher :: Fabric8 ............... SUCCESS [  6.227 s]
[INFO] Fabric8 Maven :: Enricher :: OpenShift.io .......... SUCCESS [  2.007 s]
[INFO] Fabric8 Maven :: Generator :: API .................. SUCCESS [  2.836 s]
[INFO] Fabric8 Maven :: Generator :: Java Exec ............ SUCCESS [  1.954 s]
[INFO] Fabric8 Maven :: Generator :: Spring Boot .......... SUCCESS [  1.789 s]
[INFO] Fabric8 Maven :: Generator :: Vert.x ............... SUCCESS [  2.607 s]
[INFO] Fabric8 Maven :: Generator :: Karaf ................ SUCCESS [  0.150 s]
[INFO] Fabric8 Maven :: Generator :: WildFly Swarm ........ SUCCESS [  0.127 s]
[INFO] Fabric8 Maven :: Generator :: Thorntail v2 ......... SUCCESS [  0.142 s]
[INFO] Fabric8 Maven :: Generator :: WebApp ............... SUCCESS [  1.624 s]
[INFO] Fabric8 Maven :: Watcher :: API .................... SUCCESS [  0.203 s]
[INFO] Fabric8 Maven :: Watcher :: Standard ............... SUCCESS [  0.424 s]
[INFO] Fabric8 Maven :: Plugin ............................ SUCCESS [ 17.818 s]
[INFO] Fabric8 Maven :: Enricher :: Deprecated ............ SUCCESS [  0.384 s]
[INFO] Fabric8 Maven :: Integration Tests ................. SUCCESS [  0.158 s]
[INFO] Fabric8 Maven :: Build ............................. FAILURE [ 38.649 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:03 min
[INFO] Finished at: 2019-03-02T20:48:11+00:00
[INFO] Final Memory: 113M/1219M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.6.0.1398:sonar (default-cli) on project fabric8-maven-plugin-build: The preview mode, along with the 'sonar.analysis.mode' parameter, is no more supported. You should stop using this parameter. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :fabric8-maven-plugin-build
Exited with code 1
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -82,7 +82,6 @@ jobs:
     - run: |
         if [ -n "${CIRCLE_PR_NUMBER}" ]; then
           MAVEN_OPTS="-Xmx3000m" mvn -B clean org.jacoco:jacoco-maven-plugin:prepare-agent package sonar:sonar \
-             -Dsonar.analysis.mode=preview \
              -Dsonar.github.pullRequest=${CIRCLE_PR_NUMBER} \
              -Dsonar.github.repository=fabric8io/fabric8-maven-plugin \
              -Dsonar.github.oauth=${GITHUB_COMMENT_TOKEN} \

